<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MCSL2_Dialog</class>
 <widget class="QDialog" name="MCSL2_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>提示</string>
  </property>
  <widget class="QPushButton" name="Dialog_PushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: rgb(230, 230, 230);
	border-radius: 7px;
}
QPushButton:pressed
{
	background-color: rgb(223, 223, 223);
	border-radius: 7px;
}</string>
   </property>
   <property name="text">
    <string>知道了</string>
   </property>
  </widget>
  <widget class="QLabel" name="Dialog_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>281</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
